src/file-uploader/file-uploader.js

```diff
@@ -114,7 +116,7 @@ function Unstable_FileUploader(props: PropsT) {
 
               {afterFileDrop && (
                 <React.Fragment>
-                  {!!props.progressAmount && (
+                  {props.progressAmount ? (
```

Comment:
For an initial `0` progress it will render spinner. Or is the expectation to have the progressAmount > 0?

Reply:
ah right - good catch